app/src/androidTest/java/com/github/se/travelpouch/ui/dashboard/TravelActivityScreen.kt

```diff
@@ -96,4 +97,47 @@ class TravelActivityScreen {
     composeTestRule.onNodeWithTag("activityItem").assertTextContains(activity.location.name)
     composeTestRule.onNodeWithTag("activityItem").assertTextContains("1/1/1970")
   }
+
+  @Test
+  fun verifyBannerIsDisplayedCorrectly() {
+    val nowSeconds = Timestamp.now().seconds
+    val activitiesNow =
+        listOf(
+            Activity(
+                "1",
+                "title1",
+                "description1",
+                Location(0.0, 0.0, Timestamp(0, 0), "lcoation1"),
+                Timestamp(nowSeconds + 3600L, 0),
+                mapOf<String, Int>()),
+            Activity(
+                "2",
+                "title2",
+                "description2",
+                Location(0.0, 0.0, Timestamp(0, 0), "lcoation2"),
+                Timestamp(nowSeconds + 3600L, 0),
+                mapOf<String, Int>()))
```

Comment:
Also, I was not clear enough. What would be better is to add to the variable `activitiesNow` an extra element which has an occurence date superior to 24h.

Reply:
(and also, two such that one has a date d : -24 < d < 0 and another has a date d' such that -24 > d', where d and d' are dates that happened before the date you chose)

Reply:
F I N E 😤